components/raftstore/src/store/worker/region.rs

```diff
@@ -779,6 +779,7 @@ where
             // ingested. check level 0 every time because we can not make sure
             // how does the number of level 0 files change.
             if self.ingest_maybe_stall() {
+                SNAP_INGEST_DELAY_COUNTER.inc();
```

Comment:
just add to snapshot actions instead of a new metric

Reply:
indeed, will do